### DISCUSSION TITLE
Date Picker in other languages

### DISCUSSION THREAD

**Discussion by GuilhermeHirsch** [Budibase/budibase#2720] [Help] [answered]
Hey guys! 
Is there the possibility of configuring the "date picker" component in another language or is there something planned for it? In my case it would be in Portuguese. It would make all the difference in my user experience. 
Thank you very much!

**Comment by shogunpurple** [comment, marked as answer]
Hi @GuilhermeHirsch! We have a PR up which will introduce i18n support to the builder: https://github.com/Budibase/budibase/pull/2651

We are holding off on merging it because there's a lot of testing and review to happen before something so fundamental goes into the platform. This would allow us to introduce other languages to our components. Right now we only have translations for english and chinese, but the idea would be that there will be many more in future, of course.

Obrigado e boa tarde! 😄

**Reply by GuilhermeHirsch** [reply]
@shogunpurple  support is already in Portuguese 🤣
Great news! I look forward to it then! 
Muito Obrigado! E um ótimo dia de trabalho para toda a equipe!

**Reply by dncpax** [reply]
that's great news. vamos aguardar ;)
Related to this, is it possible to have dates shown in other formats, like yyyy/mm/dd without stating the week day?

Thanks.
